sig/protos/temporal/api/enums/v1/namespace.rbs

```diff
@@ -1,4 +1,4 @@
-module Temporalio
+module Temporal
```

Comment:
Why did the namespace change on these? I think we want _everything_ in the SDK to be under `Temporalio`.

Reply:
You were too early. 😄 I should have marked that PR as a draft.